### ISSUE TITLE
Can not load pretrained models

### ISSUE THREAD

**Issue by DefTruth** [HSE-asavchenko/face-emotion-recognition#2]
```shell
 File "/Users/xxx/Library/Python/3.8/lib/python/site-packages/timm/models/efficientnet_blocks.py", line 47, in forward
    return x * self.gate(x_se)
  File "/Users/xxx/Library/Python/3.8/lib/python/site-packages/torch/nn/modules/module.py", line 947, in __getattr__
    raise AttributeError("'{}' object has no attribute '{}'".format(
AttributeError: 'SqueezeExcite' object has no attribute 'gate'
```

**Comment by HSE-asavchenko** [comment]
It seems that the versions of timm (pytorch-image-models) on my server  and yours are different… Could you clarify what is exact cell in train_emotions-pytorch.ipynb that does not work? If it is “model = torch.load(PATH)”, please, let me know the version of timm. I used latest repository cloned several months ago, so that it is possible that there are some changes there. In addition, could you verify that the following code works fine for you
model=timm.create_model('tf_efficientnet_b0_ns', pretrained=False)
model=timm.create_model('tf_efficientnet_b0_ns', pretrained=True)
?

**Comment by DefTruth** [comment]
Many thanks ~ I have solved this problem when i use timm==0.4.5 and successful reimplement the inference with c++.  

|Class|Size|From|Awesome|File|Type|State|Usage|
|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
|[EfficientEmotion7](https://github.com/HSE-asavchenko/face-emotion-recognition)|15M| [face-emo...](https://github.com/HSE-asavchenko/face-emotion-recognition) | 🔥↑ | [![](https://img.shields.io/badge/onnx-done-brightgreen.svg)](https://github.com/DefTruth/lite.ai/tree/main/docs/hub/lite.ai.hub.onnx.md#lite.ai.hub.onnx-face-attributes) | *face::attr* | ✅️ | [demo](https://github.com/DefTruth/lite.ai/blob/main/examples/lite/cv/test_lite_efficient_emotion7.cpp) & [cpp](https://github.com/DefTruth/lite.ai/blob/main/ort/cv/efficient_emotion7.cpp) |
|[EfficientEmotion8](https://github.com/HSE-asavchenko/face-emotion-recognition)|15M| [face-emo...](https://github.com/HSE-asavchenko/face-emotion-recognition) | 🔥↑ | [![](https://img.shields.io/badge/onnx-done-brightgreen.svg)](https://github.com/DefTruth/lite.ai/tree/main/docs/hub/lite.ai.hub.onnx.md#lite.ai.hub.onnx-face-attributes) | *face::attr* | ✅  | [demo](https://github.com/DefTruth/lite.ai/blob/main/examples/lite/cv/test_lite_efficient_emotion8.cpp) & [cpp](https://github.com/DefTruth/lite.ai/blob/main/ort/cv/efficient_emotion8.cpp) |
|[MobileEmotion7](https://github.com/HSE-asavchenko/face-emotion-recognition)|13M| [face-emo...](https://github.com/HSE-asavchenko/face-emotion-recognition) | 🔥↑ | [![](https://img.shields.io/badge/onnx-done-brightgreen.svg)](https://github.com/DefTruth/lite.ai/tree/main/docs/hub/lite.ai.hub.onnx.md#lite.ai.hub.onnx-face-attributes) | *face::attr* |  ✅  | [demo](https://github.com/DefTruth/lite.ai/blob/main/examples/lite/cv/test_lite_mobile_emotion7.cpp) & [cpp](https://github.com/DefTruth/lite.ai/blob/main/ort/cv/mobile_emotion7.cpp) |
|[ReXNetEmotion7](https://github.com/HSE-asavchenko/face-emotion-recognition)|30M| [face-emo...](https://github.com/HSE-asavchenko/face-emotion-recognition) | 🔥↑ | [![](https://img.shields.io/badge/onnx-done-brightgreen.svg)](https://github.com/DefTruth/lite.ai/tree/main/docs/hub/lite.ai.hub.onnx.md#lite.ai.hub.onnx-face-attributes) | *face::attr* |  ✅  | [demo](https://github.com/DefTruth/lite.ai/blob/main/examples/lite/cv/test_lite_rexnet_emotion7.cpp) & [cpp](https://github.com/DefTruth/lite.ai/blob/main/ort/cv/rexnet_emotion7.cpp)  |

**Comment by HSE-asavchenko** [comment]
Ok, thanks for letting me know and sharing your code!
Closing the issue now
